canonical-websites/docs.jujucharms.com
# Redirects for old URLs

**This PR is based on #7 - merge that first**

Add redirects for the old docs URL format. Any old versions will go to https://old-docs.jujucharms.com, and any `/devel` URLs will go to their new Discourse-based URL format for the same page.

Fixes https://github.com/ubuntudesign/base-squad/issues/375

QA
--

- `./run`
- Go to some `devel` URLs - e.g. http://0.0.0.0:8030/devel/en/credentials - check you're redirected to the new location for the same page (in this case http://0.0.0.0:8030/credentials/1112)
- Go to some older version URLs - e.g. http://0.0.0.0:8030/2.5/en/charms-relations or http://0.0.0.0:8030/2.3/en/ or http://0.0.0.0:8030/2.4. Check you're redirected to the correct page over on https://old-docs.jujucharms.com (don't worry about the invalid certificate)
